.travis.yml

```diff
@@ -31,6 +31,9 @@ jobs:
     - env: [ NAME=modules ]
       script: ./gradlew check -x testcontainers:check -x selenium:check --scan --no-daemon
 
+    - env: [ NAME=examples ]
```

Comment:
AFAIR we were running examples only on CircleCI for speed reasons

Reply:
Examples on Travis with Maven was PITA because of snapshot dependency and strange race conditions.

I'd argue, we can risk to enable them now with composite build.